lib/src/bezier_chart_widget.dart

```diff
@@ -151,26 +151,24 @@ class BezierChartState extends State<BezierChart>
   List<double> _tempYValues;
 
   ///Refresh the position of the vertical/bubble
-  _refreshPosition(details) {
+  void _refreshPosition(details) {
     if (_animationController.status == AnimationStatus.completed &&
         _displayIndicator) {
-      _updatePosition(details);
+      return _updatePosition(details.globalPosition);
```

Comment:
why `return` if this is a `void` function and there is no more code after the `if` condition

Reply:
This avoids to use more callstack. Here I explain how it works but in Spanish: https://medium.com/@kranfix/optimizaci%C3%B3n-de-la-pila-de-llamadas-en-funciones-recursivas-796b3286ecd4